src/components/notifications/MobivateSMS.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+    <div class="mb-3">
+        <label for="mobivate-key" class="form-label">{{ $t("API Key") }}<span style="color: red;"><sup>*</sup></span></label>
+        <HiddenInput id="mobivate-key" v-model="$parent.notification.mobivateApikey" :required="true" autocomplete="new-password"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <label for="mobivate-recipients" class="form-label">{{ $t("Recipients") }}</label>
+        <input id="mobivate-recipients" v-model="$parent.notification.mobivateRecipients" type="text" minlength="3" maxlength="20" pattern="^[\d+,]+$" class="form-control" required>
+        <div class="form-text">
+            <p>{{ $t("Comma separated list of numbers in international format. (eg. 447930000000,447930000001)") }}</p>
```

Comment:
Please make sure that all translations are in en.json as otherwise they cannot be translated